product-service/package.json

```diff
@@ -5,6 +5,7 @@
   "main": "serverless.ts",
   "scripts": {
     "deploy": "sls deploy",
+    "deploy catalogBatchProcess": "serverless deploy function -f catalogBatchProcess",
```

Comment:
`"deploy:function": "sls deploy function -f" `
you can use 
`npm run deploy:function catalogBatchProcess`